python/dgl/nn/pytorch/conv/gatconv.py

```diff
@@ -208,10 +210,12 @@ def reset_parameters(self):
             nn.init.xavier_normal_(self.fc_dst.weight, gain=gain)
         nn.init.xavier_normal_(self.attn_l, gain=gain)
         nn.init.xavier_normal_(self.attn_r, gain=gain)
-        if self.bias is not None:
+        if hasattr(self, "bias"):
             nn.init.constant_(self.bias, 0)
         if isinstance(self.res_fc, nn.Linear):
             nn.init.xavier_normal_(self.res_fc.weight, gain=gain)
+            if self.res_fc.bias is not None:
```

Comment:
not needed anymore

Reply:
Why isn't it needed anymore? We still can have both situations when fc has no or has bias inside, which might need to be reset.

Reply:
I see you. You are right.